onnx/onnx_pb.h

```diff
@@ -50,11 +50,13 @@
 // issue, and for anyone dependent on ONNX it will be defined as
 // ONNX_IMPORT.
 // This is a solution from https://github.com/caffe2/caffe2/blob/4f534fad1af9f77d4f0496ecd37dafb382330223/caffe2/core/common.h
+#ifndef ONNX_API
 #ifdef ONNX_BUILD_MAIN_LIB
```

Comment:
Maybe we should define `ONNX_BUILD_MAIN_LIB` if it's a dynamic lib, otherwise we don't define it.

Reply:
That would be right to do, I did just a minimal fix